pipelines/hacbs/hacbs-test.yaml

```diff
@@ -22,11 +22,23 @@
       name: sanity-label-check
     params:
     - name: IMAGE_URL
+      value: $(params.output-image)   
```

Comment:
you removed workspace to sanity-label-check

Reply:
Sorry for that, corrected.